### ISSUE TITLE
[Feature] IPage to support getting the cookies applicable to the open page

### ISSUE THREAD

**Issue by StevenDenman** [microsoft/playwright-dotnet#1924]
Currently the only way to get the cookies is to use the browser context and filtering out the domain.

It would be nice if the IPage had a CookiesAsync function that automatically returned the cookies applicable to the Page.Url value so that the tests only need an IPage and don't need to also track the IBrowserContext to get them

**Comment by mxschmitt** [comment]
You can access the browser context via `page.Context` so something like this should be possible:

```csharp
page.Context.CookiesAsync(new[] { page.Url });
```

**Comment by StevenDenman** [comment]
Yes I can implement that as an extension method but it would be nice if there was a page.CookiesAsync() method that does that in the background?
